tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/cocosnet_evaluator.py

```diff
@@ -0,0 +1,532 @@
+        raise NotImplementedError
+
+    def release(self):
+        del self.network
+        del self.exec_network
+
+    def load_model(self, network_info, launcher, log=False):
+        model, weights = self.auto_model_search(network_info, self.net_type)
+        if weights:
+            self.network = launcher.read_network(model, weights)
+            self.exec_network = launcher.ie_core.load_network(self.network, launcher.device)
+        else:
+            self.network = None
+            self.exec_network = launcher.ie_core.import_network(str(model))
+        self.set_input_and_output()
+        if log:
+            self.print_input_output_info()
+
+    def load_network(self, network, launcher):
+        self.network = network
+        self.exec_network = launcher.ie_core.load_network(self.network, launcher.device)
+        self.set_input_and_output()
+
+    def set_input_and_output(self):
+        pass
+
+    def print_input_output_info(self):
+        print_info('{} - Input info:'.format(self.net_type))
+        has_info = hasattr(self.network if self.network is not None else self.exec_network, 'input_info')
+        if self.network:
+            if has_info:
+                network_inputs = OrderedDict(
+                    [(name, data.input_data) for name, data in self.network.input_info.items()]
+                )
+            else:
+                network_inputs = self.network.inputs
+            network_outputs = self.network.outputs
+        else:
+            if has_info:
+                network_inputs = OrderedDict([
+                    (name, data.input_data) for name, data in self.exec_network.input_info.items()
+                ])
+            else:
+                network_inputs = self.exec_network.inputs
+            network_outputs = self.exec_network.outputs
+        for name, input_info in network_inputs.items():
+            print_info('\tLayer name: {}'.format(name))
+            print_info('\tprecision: {}'.format(input_info.precision))
+            print_info('\tshape {}\n'.format(input_info.shape))
+        print_info('{} - Output info'.format(self.net_type))
+        for name, output_info in network_outputs.items():
+            print_info('\tLayer name: {}'.format(name))
+            print_info('\tprecision: {}'.format(output_info.precision))
+            print_info('\tshape: {}\n'.format(output_info.shape))
+
+
+class CorrespondenceNetwork(BaseModel):
+    def __init__(self, network_info, launcher, delayed_model_loading=False):
+        self.net_type = "correspondence_network"
+        super().__init__(network_info, launcher, delayed_model_loading)
+
+    def set_input_and_output(self):
+        has_info = hasattr(self.exec_network, 'input_info')
+        if has_info:
+            self.inputs = OrderedDict([(name, data.input_data) for name, data in self.exec_network.input_info.items()])
+        else:
+            self.inputs = self.exec_network.inputs
+        self.outputs = self.exec_network.outputs
```

Comment:
inputs and outputs store runtime info from network, they also should be removed before network.
It may be the reason of segfault

Reply:
It was the reason of segfault. Thanks